internal/daemonset/daemonset.go

```diff
@@ -405,46 +405,39 @@ func makeLoadCommand(inTreeModuleToRemove string, spec kmmv1beta1.ModprobeSpec,
 	var loadCommand strings.Builder
 
 	if inTreeModuleToRemove != "" {
-		loadCommand.WriteString("modprobe -r")
-		if spec.DirName != "" {
-			loadCommand.WriteString(" -d " + spec.DirName)
-		}
-		loadCommand.WriteString(" " + inTreeModuleToRemove + " && ")
+		fmt.Fprintf(&loadCommand, "modprobe -r %q && ", inTreeModuleToRemove)
 	}
 
 	if fw := spec.FirmwarePath; fw != "" {
-		fmt.Fprintf(&loadCommand, "cp -r %s/* %s && ", fw, nodeVarLibFirmwarePath)
+		fmt.Fprintf(&loadCommand, `cp -r "%s/*" %s && `, fw, nodeVarLibFirmwarePath)
```

Comment:
what about nodeVarLibFirmwarePath? Don't we want to quote it also?

Reply:
you are right, then we don't need them in line 458

Reply:
We do because we concatenate it with file names present in the container image's filesystem, and those are user-controlled strings.